janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/astyanax/AstyanaxKeyColumnValueStore.java

```diff
@@ -432,6 +432,7 @@ public void remove() {
             };
         }
 
+                
```

Comment:
Could you remove all independent file changes?

Reply:
From the pr

Reply:
Hi @farodin91 , Thank you for the feedback. I checked in my changes with your review feedback.